src/main/java/com/google/api/generator/gapic/composer/ServiceClientCommentComposer.java

```diff
@@ -162,16 +162,18 @@ static CommentStatement createCreateMethodStubArgComment(
   }
 
   static List<CommentStatement> createRpcMethodHeaderComment(
-      Method method, List<MethodArgument> methodArguments) {
+      Method method, List<MethodArgument> methodArguments, String sampleCode) {
     JavaDocComment.Builder methodJavadocBuilder = JavaDocComment.builder();
 
     if (method.hasDescription()) {
       methodJavadocBuilder =
           processProtobufComment(method.description(), methodJavadocBuilder, null);
     }
 
-    // methodJavadocBuilder.addParagraph(METHOD_DESCRIPTION_SAMPLE_CODE_SUMMARY_STRING);
-    // TODO(summerji): Add sample code here and uncomment the above.
+    if (!sampleCode.isEmpty()) {
```

Comment:
Prefer using `Strings.isNullOrEmpty` instead, please see existing examples in the codebase.

Reply:
SG